### PR TITLE
[👶 PR] Handle logger notes in wandb

### PR DESCRIPTION
**Pull request by vict0rsch**
Add a `notes:` option for the logger to serve as description to the wandb run (defaults to `None`)

(From #224)

### CODE REVIEW

**Review by alexhernandezgarcia — APPROVED** [review]
Without having tried it yet, it seems correct!

File: gflownet/utils/logger.py [hunk 1]
```diff
@@ -60,7 +61,7 @@ def __init__(
             if slurm_job_id:
                 wandb_config["slurm_job_id"] = slurm_job_id
             self.run = self.wandb.init(
-                config=wandb_config, project=project_name, name=run_name
+                config=wandb_config, project=project_name, name=run_name, notes=notes
```
Comment (alexhernandezgarcia) [inline gflownet/utils/logger.py:64]: So this goes on the description section of the wandb overview, where it says something like "What is special about this run?"?

Comment (vict0rsch) [reply gflownet/utils/logger.py:64]: Yes exactly

SCR-20230925-oovt

File: mila/launch.py [hunk 2]
```diff
@@ -542,13 +568,20 @@ def print_md_help(parser, defaults):
             sbatch_path.parent.mkdir(parents=True, exist_ok=True)
             # write template
             sbatch_path.write_text(templated)
-            print(f"  🏷  Created ./{sbatch_path.relative_to(Path.cwd())}")
+            print()
```
Comment (carriepl) [inline mila/launch.py:571]: Why keep the print if there is nothing to print?

**Review by carriepl — COMMENTED** [review]
Looks generally good to me, though I didn't run the code and I am not super familiar with the launcher tool. I only added tiny question that would take all of 15 seconds to address.

**Pull request merged**